c_reference/include/dscnn.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#ifndef __DSCNN__
+#define __DSCNN__
+
+/**
+ * @brief Model definition for the 1D Convolution sub-block applied before the RNN
+ * @brief sub-layers : BatchNorm1d -> Conv1D_LR
+ * 
+ * @param[out]   output_signal       pointer to the final output signal, minimum size = out_T * in_channels. out_T has to be calculated based on the reduction from all the conv and pool layers
+ * @param[in]    input_signal        pointer to the input signal. size = in_T * in_channels
+ * @param[in]    in_T                number of time steps in the input
+ * @param[in]    in_channels         number of input channels. The output will have the same number of channels
+ 
+ * @param[in]    mean                pointer to the mean for the batch normalization, size = in_channels
+ * @param[in]    var                 pointer to the variance for the batch normalization, size = in_channels
+ * @param[in]    affine              whether the affine operations are applied
+ * @param[in]    gamma               pointer to the scaling factors for the post-norm affine operation, size = in_channels
+ * @param[in]    beta                pointer to the scalar offsets for the post-norm affine operation, size = in_channels
+ * @param[in]    in_place            in place computation of the batchnorm. Storage efficient
+ * 
+ * @param[in]    cnn_hidden          hidden state/out_channels dimensions for the CNN
+ * @param[in]    cnn_padding         padding for the CNN layer. Note: applied to both sides of the input 
+ * @param[in]    cnn_kernel_size     kernel size of the CNN
+ * @param[in]    cnn_params          weights, bias and other essential parameters used to describe the CNN
+ * @param[in]    cnn_activations     an integer to choose the type of activation function.
+ *                                   0: none
+ *                                   1: sigmoid
+ *                                   2: tanh
+ *                                   3: relu
+ */
+int dscnn_lr(float* output_signal, float* input_signal, unsigned in_T, unsigned in_channels,
```

Comment:
can we think of better names for both functions in this file (before and after)

Reply:
I've renamed the functions to phon_pred_lr_cnn and phon_pred_depth_point_lr_cnn